### ISSUE TITLE
the plugin 9.41 causing the app to crash immediately

### ISSUE THREAD

**Issue by mobeenk** [playgameservices/play-games-plugin-for-unity#1974]
I've been trying to find a solution for this since 2 weeks
I'm using Unity v5.4 with this plugin
this is the code 
    void Start()
    {
        PlayGamesClientConfiguration config = new PlayGamesClientConfiguration.Builder().Build();
        PlayGamesPlatform.InitializeInstance(config);
        PlayGamesPlatform.DebugLogEnabled = true;
        PlayGamesPlatform.Activate();
       signin();
    public void signin()
    {
       PlayGamesPlatform.Instance.Authenticate
      //  Social.localUser.Authenticate
       ((bool success) => {
            if (success)
            {
                Debug.Log("yes");
            }
            else if (!success)
            {
                Debug.Log("no");
            }
        });
    }

the crash happened at the signin() part 
so basically authinticating is crashing 
I tried the solution in here 
https://github.com/playgameservices/play-games-plugin-for-unity/issues/1604
it didn't work
I also change the SHA-1 certificate and modified the androidManifest file still nothing worked out

Can someone please tell me what's the solution for this I'm desperate !!!

**Comment by xenonsin** [comment]
#1935 

Check this out. It seems that in 9.41 Unity isn't picking up the google generated AndroidManifest file so it omits the AppID from the final AndroidManifest. You can verify this by Analyzing your apk using Android Studio.

**Comment by mobeenk** [comment]
What the solution for this?
when I downgraded to plugin version 9.33 I get no crash and I can access leaderboards and achievments but the code for making progress doesn't work why?

**Comment by xenonsin** [comment]
If you follow the thread, there is an error in 9.41 where Unity autogenerates an AndroidManifest without an AppID, so you're going to have authentication errors. [The solution is to override the manifest so it contains the app id.](https://docs.unity3d.com/Manual/android-manifest.html)

**Comment by blookmaker** [comment]
Hi i was struggling with the exact same problem with the exact same version of unity3D v5.4 , i have just switched to 5.6 and guess what ? it's work now , if you don't want to struggle with manifests files , just switch to 5.6 ,

**Comment by vaibhavgpt-google** [comment]
Closing this issue as part of a cleanup of older tickets. Google Play Games v1 is now deprecated. Please upgrade to version v2.1.0 or newer: https://github.com/playgameservices/play-games-plugin-for-unity/releases/tag/v2.1.0. If this issue persists on the new v2 architecture, please open a new ticket.
